openrr-nav-viewer/src/bevy_app.rs

```diff
@@ -456,13 +456,7 @@ fn ui_system(
                 .add_sized([200., 30.], egui::Button::new("Reload planner config"))
                 .clicked()
             {
-                let mut planner = res_nav.planner.lock();
-                planner
-                    .update_params_from_config(format!(
```

Comment:
Given that the `update_params_from_config` is no longer in use and that it is inefficient because it would be updating while holding locks, it would be better to remove the `update_params_from_config`.